10-inference.Rmd

```diff
@@ -0,0 +1,806 @@
+    group_by(flavour) %>% 
+    summarize(n = n(),
+             proportion = n() / 10000)
+```
+
+Suppose we buy a box of 40 randomly-selected Timbits and count the number of chocolate Timbits,
+i.e., take a random sample of size 40 from our Timbits population. The function
+`rep_sample_n` from the `infer` package will allow us to sample. The arguments
+of `rep_sample_n` are (1) the data frame to sample from, and (2) the size of the sample
+to take.
+
+```{r 11-example-proportions3, echo = TRUE, message = FALSE, warning = FALSE}
+samples_1 <- rep_sample_n(tbl = virtual_box, size = 40)
+choc_sample_1 <- summarize(samples_1, n = sum(flavour == "chocolate"),
+                                        prop = sum(flavour == "chocolate") / 40)
+choc_sample_1
+```
+Here we see that the proportion of chocolate Timbits in this random sample is
+`r round(choc_sample_1$prop,2)`. This value is our sample statistic; since it is a single
+value that is used to estimate a population parameter, we refer to it as a **point estimate**.
+
+Now imagine we took another random sample of 40 Timbits from the population. Do you
+think you would get the same proportion? Let's try sampling from the population
+again and see what happens. 
+
+```{r 11-example-proportions4, echo = TRUE, message = FALSE, warning = FALSE}
+set.seed(2)
+samples_2 <- rep_sample_n(virtual_box, size = 40)
+choc_sample_2 <- summarize(samples_2, n = sum(flavour == "chocolate"),
+                                        prop = sum(flavour == "chocolate") / 40)
+choc_sample_2
+```
+
+Notice that we get a different value for our statistic this time. The
+proportion of chocolate Timbits in this sample is `r round(choc_sample_2$prop, 2)`. 
+If we were to do this again, another random sample could also give a
+different result. Statistics vary from sample to sample 
+due to **sampling variability**. 
+
+But just how much should we expect the statistics of our random
+samples to vary? In order to understand this, we will simulate more samples
+of size 40 from our population of Timbits, and calculate the 
+proportion of chocolate Timbits in each sample. We can then
+construct the distribution of sample proportions we calculate. The distribution
+of the statistic for all possible samples of size $n$ from a population is
+called a **sampling distribution**. The sampling distribution will help us see
+how much we would expect our sample proportions from this population to vary
+for samples of size 40. Below we again use the `rep_sample_n` to take samples
+of size 40 from our population of Timbits, but we set the `reps` argument
+to specify the number of samples to take.
+
+```{r 11-example-proportions5, echo = TRUE, message = FALSE, warning = FALSE}
+samples <- rep_sample_n(virtual_box, size = 40, reps = 1000)
+head(samples)
+tail(samples)
+```
+
+Notice the column `replicate` is indicating the replicate with which each
+Timbit belongs. Since we took 1000 samples of size 40, there are 1000 replicates.
```

Comment:
```suggestion
Notice the column `replicate` is indicating the replicate, or sample, with which each
Timbit belongs. Since we took 1000 samples of size 40, there are 1000 replicates.
```